during.md

```diff
@@ -100,37 +107,42 @@
   [GitPod](https://gitpod.io/) may offer a way for participants to contribute to the lesson
   without the need for setting up a local development environment.
   As with the "primer" session mentioned above,
-  this skill-up may fit best at the beginning of the sprint,
-  before the first co-working session begins, or as a separate event
+  this skill-up may fit best at the beginning of the sprint.
+  This can also run as a separate event
   that takes place in the days leading up to the full sprint.
-- **Flag pull requests as draft and ready for review as you work.**
+
+- **Flag pull requests as a draft and ready for review as you work.**
   Opening draft pull requests early, while work on a task is ongoing,
   can help contributors understand where progress is being made and identify
   tasks/issues that still need to be tackled.
   Marking pull requests as ready for review helps maintainers prioritise their
   focus during the sprint and maintains momentum by ensuring that completed work
   is merged into the lesson as soon as possible.
-  Opening draft pull requests is particularly important at the end of a sprint,
+  Opening draft pull requests are particularly important at the end of a sprint,
```

Comment:
```suggestion
  Opening draft pull requests is particularly important at the end of a sprint,
```

Reply:
I see I made a lot wrong edits :D Thank you for being patient with this.